edge/services/nginx-operator/horizon/pattern.json

```diff
@@ -2,7 +2,7 @@
     "name": "pattern-${SERVICE_NAME}-$ARCH",
     "label": "Edge $SERVICE_NAME Service Pattern for $ARCH",
     "description": "Pattern for $SERVICE_NAME for $ARCH",
-    "public": true,
+    "public": false,
```

Comment:
Why is the pattern not public? Don't we want users to be able to try it from their own org, referencing this pattern in the IBM org?